src/locales/en/translation.json

```diff
@@ -296,8 +296,8 @@
         },
         "formErrorMessage": {
           "errorSummaryTitleText": "There is a problem",
-          "errorSummaryDescriptionText": "Select what you would like to do",
-          "errorRadioMessage": "Select what you would like to do"
+          "errorSummaryDescriptionText": "Select how you would like to prove your identity",
+          "errorRadioMessage": "Select how you would like to prove your identity"
```

Comment:
Hmm I thought the change for this template would be for [PYIC-7211](https://govukverify.atlassian.net/browse/PYIC-7211)?